### ISSUE TITLE
Fn key is not identified on Trust Bluetooth Keyboard

### ISSUE THREAD

**Issue by plato79** [pqrs-org/Karabiner-Elements#2436]
I have a BT keyboard which is similar to magic keyboard.. Though some features are not same.. For example it uses TR layout but some keys are not where they're originally.. For example double quote is normally on the corner of the keyboard, just below ESC key. But on this keyboard there's "< > |" key.. 

Well, the problem is they assigned double quote key to CTRL+ALT+"İ" key.. This sometimes works, but sometimes not..

Also as far as I see the Fn key is just not detected with your app.. F1..F12 keys have all these functions and it's by default. So if you press F12 it executes volume up instead of F12 key.. Clicking switch functionality of F.. keys doesn't work.

Also I tried some rules with Fn key in it.. Nothing seems to work.

Do you know any way to program this? I tried EventViewer.. While it shows the key, as far as I see it doesn't show anything about Fn key.

Here is the hidutil -list output:

```
VendorID ProductID LocationID UsagePage Usage RegistryID  Transport Class                     Product                               UserClass                 Built-In 
0x5ac    0x220     0x50000ea  1         6     0x10000072c Bluetooth IOBluetoothHIDDriver      Trust Bluetooth Keyboard              (null)                    1
```

**Comment by mehdico** [comment]
I have the same problem with this model. did you find any solution?
@plato79

**Comment by plato79** [comment]
Not really, sorry. I tried different things but couldn't find a solution.
